app/components/invite-card.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useFetcher } from "react-router";
+import { CopyButton } from "./copy-button";
+import { Input } from "./input";
+
+type InviteCardProps = {
+	setShowInvite: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
could easily be


```suggestion
	setShowInvite: (show: boolean) => void;
```

Plus this is wrong API naming. From how it's used, you're only attempting to close the invite card.

A better design is to say:

```
onClose: VoidFunction
```

then whereever it is used, you say

```tsx
onClose={() => setSomething(false)}
```